chaincfg/init_test.go

```diff
@@ -179,21 +179,21 @@ var (
 				Id:          "Abstain",
 				Description: "Abstain voting for Pedro",
 				Bits:        0x0, // 0b0000
-				IsIgnore:    true,
+				IsAbstain:   true,
 				IsNo:        false,
 			},
 			{
 				Id:          "Yes",
 				Description: "Vote for Pedro",
 				Bits:        0x2,  // 0b0010
-				IsIgnore:    true, // XXX this is the invalid choice
+				IsAbstain:   true, // XXX this is the invalid choice
```

Comment:
This reads more like setting 0x2 as the valid option to indicate abstaining.  Don't know why that would ever be used.